e2e/process-services/start-process-component.e2e.ts

```diff
@@ -60,402 +72,485 @@ describe('Start Process Component', () => {
+                await processFiltersPage.clickNewProcessDropdown();
+                await startProcessPage.enterProcessName('Audit Log');
+                await startProcessPage.selectFromProcessDropdown(processModelWithSe);
+                await startProcessPage.clickFormStartProcessButton();
+                await processFiltersPage.clickRunningFilterButton();
+                await processFiltersPage.selectFromProcessList('Audit Log');
+                await processDetailsPage.clickAuditLogButton();
+
+                await expect(await FileBrowserUtil.isFileDownloaded(auditLogFile)).toBe(true);
+            });
+
+            it('Should be able to attach a file using the button', async () => {
+                await processServicesPage.goToApp(app.title);
+
+                await processServiceTabBarPage.clickProcessButton();
+
+                await processFiltersPage.clickCreateProcessButton();
+                await processFiltersPage.clickNewProcessDropdown();
+
+                await startProcessPage.enterProcessName('Attach File');
+                await startProcessPage.selectFromProcessDropdown(processModelWithSe);
+                await startProcessPage.clickFormStartProcessButton();
+
+                await processFiltersPage.clickRunningFilterButton();
+                await processFiltersPage.selectFromProcessList('Attach File');
+
+                await attachmentListPage.clickAttachFileButton(jpgFile.location);
+                await attachmentListPage.checkFileIsAttached(jpgFile.name);
+            });
+
+            it('[C260451] Should be possible to display process diagram', async () => {
+                await processServicesPage.goToApp(app.title);
+
+                await processServiceTabBarPage.clickProcessButton();
+
+                await processFiltersPage.clickCreateProcessButton();
+                await processFiltersPage.clickNewProcessDropdown();
+
+                await startProcessPage.enterProcessName('Show Diagram');
+                await startProcessPage.selectFromProcessDropdown(processModelWithSe);
+                await startProcessPage.clickFormStartProcessButton();
+
+                await processFiltersPage.clickRunningFilterButton();
+                await processFiltersPage.selectFromProcessList('Show Diagram');
+
+                await processDetailsPage.clickShowDiagram();
+            });
+
+            it('[C260452] Should redirect user when clicking on active/completed task', async () => {
+                await processServicesPage.goToApp(app.title);
+                await processServiceTabBarPage.clickProcessButton();
+                await processFiltersPage.clickCreateProcessButton();
+                await processFiltersPage.clickNewProcessDropdown();
+                await startProcessPage.enterProcessName('Active Task');
+                await startProcessPage.selectFromProcessDropdown(processModelWithSe);
+                await startProcessPage.clickFormStartProcessButton();
```

Comment:
These 5 lines are repeated in many places to create a process and type the process name and select the processDefinition. Consider if we can put these lines into a method called createProcess(processName: string, processDefinition: string) or something similar in this way we can reduce repeating the same steps and the code can be more readable.